pkg/flypg/node.go

```diff
@@ -234,10 +270,98 @@ func (n *Node) PostInit(ctx context.Context) error {
 		if member != nil && member.Role != "" {
 			role = member.Role
 		}
+		fmt.Printf("My current role is: %s\n", role)
 
 		switch role {
 		case PrimaryRoleName:
-			// TODO - This is where we need to fence the primary if it's no longer valid.
+			standbys, err := repmgr.StandbyMembers(ctx, conn)
+			if err != nil {
+				if !errors.Is(err, pgx.ErrNoRows) {
+					return fmt.Errorf("failed to query standbys")
+				}
+			}
+
+			totalMembers := len(standbys) + 1 // include self
+			totalActive := 1                  // include self
+			totalInactive := 0
+			totalConflicts := 0
+			conflictMap := map[string]int{}
+
+			for _, standby := range standbys {
+				mConn, err := repmgr.NewRemoteConnection(ctx, standby.Hostname)
+				if err != nil {
+					fmt.Printf("failed to connect to %s", standby.Hostname)
+					totalInactive++
+					continue
+				}
```

Comment:
Should these `mConn` connections be closed? It doesn't look like they're used after passing to `PrimaryMember()`.

Reply:
Yes they should!